libs/plugins/flow-client/src/lib/triggers/triggers.module.ts

```diff
@@ -5,7 +5,7 @@ import { CoreModule as FlogoCoreModule } from '@flogo-web/lib-client/core';
 import { SharedModule as FlogoSharedModule } from '@flogo-web/lib-client/common';
 
 import { CoreModule as FlowCoreModule } from '../core';
-import { InstallerModule as ContribInstallerModule } from '../shared/installer';
+import { InstallerModule as ContribInstallerModule } from '@flogo-web/lib-client/installer';
```

Comment:
I've noticed we are importing `InstallerModule` as `ContribInstallerModule` everywhere, we should probably just rename the module to `ContribInstallerModule` and the package as well `@flogo-web/lib-client/contrib-installer`.

Reply:
Done.